src/script.js

```diff
@@ -161,6 +176,7 @@ function isCanonicalSignature (buffer) {
 module.exports = {
   compile: compile,
   decompile: decompile,
+  toWitness: toWitness,
```

Comment:
> https://github.com/bitcoin/bitcoin/blob/fc4f4547b7f6ea13d273ad926738fcdffee36295/src/script/interpreter.cpp#L1352
> Witness data is NOT script. https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki#specification

In normal scripts, you arrive at `stack` by interpreting a scriptSig. Witness data is basically the stack, ready to be operated on by the witness script. So, if you're parsing signatures, if it's P2SH, a regular p2pk/p2pkh/multisig, you have to decompile scriptSig and do some work. 

But if it's witness, you've already got the Buffers ready to parse, and the layout of the element (`sig pubkey` for p2pkh, `sig` for P2PK, `OP_0 sig sig` for multisig) is exactly the same. 

It's because witnesses don't have opcodes that I'd like a function that has the `script` -> `push only elements` semantics. It has useful side effects when trying to extract sigs/sign/serialize signatures.